Misc/NEWS.d/next/Library/2023-10-12-18-19-47.gh-issue-82300.P8-O38.rst

```diff
@@ -0,0 +1 @@
+Add ``track`` parameter to :class:`multiprocessing.shared_memory.SharedMemory` that allows using shared memory blocks without having to register with the resource tracker.
```

Comment:
Perhaps "POSIX resource tracker that automatically releases them upon process exit" rather than "resource tracker". So that people reading this not intimately familiar with the problem already know more about who would want this and why.